README.md

```diff
@@ -1,3 +1,38 @@
-# Blackhole.service for DEIS
+# Blackhole service for Deis
 
-This will be a repository for dockerized blackhole service for DEIS.
+A repository for a docker container with simple Blackhole implementation for Deis (CoreOS) platform.
+
+The script bans every failed attempt to login by password on CoreOS instances except ip addresses that are included in `WHITELIST` variable.
```

Comment:
I would rewrite it as:

The service will ban every ip address that tried to login onto an instance with a wrong password. Except for those that are listed in the `WHITELIST` variable.